pkg/operator/ceph/cluster/watcher.go

```diff
@@ -516,7 +521,7 @@ func (c *clientCluster) createNetworkFence(ctx context.Context, pv corev1.Persis
 
 	networkFence := &addonsv1alpha1.NetworkFence{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      node.Name,
+			Name:      node.Name + "-" + driver,
```

Comment:
Since this name generation is used in several places, how about a helper method to implement it? Something like:
```suggestion
			Name:      fenceResourceName(node.Name, driver),
```